bundle-size/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@mui/bundle-size",
```

Comment:
Per https://www.notion.so/mui-org/engineering-mui-utils-purpose-9a9fc9da3a004864b6c4e1f4d1f24f95?pvs=4#101cbfe7b660800f9d59c9204ece4ec7?

```suggestion
  "name": "@mui/internal-bundle-size",
```

Reply:
This is not publishable, nor a dependency of other packages. it's only here to allow us to have it able to declare its own dependencies. I think the naming doesn't apply, or we should create a new category of naming. In testing there are more such packages, e.g. to be able have typescript tests run on build artifacts instead of aliases. We've been naming some of those with `@mui/test-`. How do you feel about this?

Reply:
Ok, I'm just removing the name altogether, problem solved 🙂

Reply:
> https://www.notion.so/mui-org/engineering-mui-utils-purpose-9a9fc9da3a004864b6c4e1f4d1f24f95?pvs=4#d1699bb48f0f4ef58ca831d1ee9772ec

Ah right, ok, my bad, I agree, it's a different class. I propose this then: 

<img width="703" alt="SCR-20250508-uatk" src="https://github.com/user-attachments/assets/e49b37d8-bdf9-436b-a1bb-0dadb1b819cc" />

https://www.notion.so/mui-org/engineering-mui-utils-purpose-9a9fc9da3a004864b6c4e1f4d1f24f95?pvs=4#d1699bb48f0f4ef58ca831d1ee9772ec

> I'm just removing the name altogether, problem solved 

Oh yeah, if it's not depended on, even better 👍

Reply:
> this would become `@mui/private-chat-uikit`.

If it's marked as private in package.json, why does it need another marker in the name? I don't see the benefit, the goal with internal is to signal something to people that want to install it, but private packages aren't published in the first place.